### ISSUE TITLE
Cannot move a military unit to a tile with a civilian unit without declaring war

### ISSUE THREAD

**Issue by axatin** [LoneGazebo/Community-Patch-DLL#9508]
Version 3.0.4, non-EUI

Moving a military unit to a tile occupied by a civilian unit of another player triggers a pop-up to declare war. But the two unit would be able to stack. This is an unnecessary limitation of movement options.
![Screenshot (195)](https://user-images.githubusercontent.com/95587882/205295103-e4f4ad1a-8ce8-4cbe-8f3e-944f7f18e82c.png)
![Screenshot (196)](https://user-images.githubusercontent.com/95587882/205295114-36ba2594-2402-451f-8729-91c4f62830fa.png)
[Savegame](https://github.com/LoneGazebo/Community-Patch-DLL/files/10141157/MoveOnCivilianNotDeclareWar.zip)

**Comment by ilteroi** [comment]
you must be american, it's all about freedom :)

switch the UI to move mode and you're good

**Comment by LessRekkless** [comment]
Right-click defaults to attack if you're using a ranged unit and targeting a tile with a unit in it.

**Comment by AndreyK-555** [comment]
Also this is the case when neutral unit (civilian or embarked on water) occupies the tile with enemy unit - you just can't attack it.
